frontend/src/components/SchemaEditor/Panels/DatabaseEditor.vue

```diff
@@ -248,6 +278,12 @@ const tableHeaderList = computed(() => {
   ];
 });
 
+onMounted(() => {
+  if (schemaList.length > 0) {
+    state.selectedSchema = head(schemaList)?.name || "";
+  }
+});
```

Comment:
I believe that `head` in lodash-es returns `undefined` while `schemaList` is empty. So we don't need the preflight check `schemaList.length > 0`

Reply:
Updated